HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -1421,6 +1421,150 @@ Patch "Always Allow Boost to Damage Objects" in "Gameplay" by "Hyper" does "Allo
     Sonic.State.AddStateNotifyAction(NotifyStateRunDamageCollision);
 }
 
+Code "Always Allow Crouching on Rails" in "Gameplay" by "WasifBoomz" does
```

Comment:
This name doesn't really make sense as Sonic never had the ability to crouch on rails in this game.

Please rename it to something else. Codes that add gameplay features don't need to follow naming conventions from other codes (see `Super Stomp`).